.github/workflows/docker-build-validation.yml

```
@@ -4,9 +4,15 @@ on:
   pull_request:
     branches:
       - main
+    paths-ignore:
+      - helm-chart/**
+      - infra/**
   merge_group:
     branches:
       - main
+    paths-ignore:
```

actionlint (1.7.7)

"paths-ignore" filter is not available for merge_group event. it is only for push, pull_request, pull_request_target events

(events)